### DISCUSSION TITLE
Select/unselect all issues in TE3 as in the daxoptimizer web app

### DISCUSSION THREAD

**Discussion by ricky7uio** [TabularEditor/TabularEditor3#1397] [Ideas]
**_The feature request_**

I would like to be able to check/uncheck all issues in TE3 as it's possible in the web app:

![image](https://github.com/user-attachments/assets/b8d0235b-2936-4016-a8dc-3b8662ef7b39)
![image](https://github.com/user-attachments/assets/0f00df5e-acee-4cbd-84c9-4b0a1dbcfa73)

In big measures, this makes easier to check the affected row/lines without all the measure code being highlighted at the same time.

**_Proposed solution_**

Add the same functionally to be able to unselect/select all issues.
Thank you

**Comment by mlonsk** [comment]
Hi @ricky7uio 
Thank you for the suggestions. I can definitely see the need for it, and we will see what we can do about including it in one of the coming releases.

**Reply by ricky7uio** [reply]
that would be great, thank you!
